OpenOversight/app/templates/partials/officer_descriptions.html

```diff
@@ -8,7 +8,9 @@ <h3>Descriptions</h3>
           {{ paragraph | safe }} <br/>
         {% endfor %}
         <br />
-        <em>{{ description.creator.username }} </em>
+        {% if not current_user.is_anonymous %}
```

Comment:
It seems like in a couple places we do `if current_user and current_user.is_???`, do we need to do a none check here?

Reply:
Oh, good call! I'll add that to keep it consistent at least.